test/e2e/network/netpol/kubemanager.go

```diff
@@ -22,6 +22,7 @@ import (
 	"net"
```

Comment:
ignore my comments if they dont apply, just wanna make sure were being as explicit as possible on exponential backoff ... didnt get time to read the code deeply.